tests/CounterInShopingBag.spec.ts

```diff
@@ -0,0 +1,82 @@
+import {test, expect} from '@playwright/test'
+import {NavigationPage } from '../page-objects/navigationPage'
+import {BeddingPage} from '../page-objects/beddingPage'
+
+test.beforeEach(async ({page})=>{
+    
+    await page.goto('https://www.shopist.io/')
+
+})
+    
+    test('Counter in shopping bag', async ({ page }) => {
+        
+        const navigateTo = new NavigationPage(page);
+        const onBeddingPage = new BeddingPage(page);
+
+        //navigate to Shop Now page 
+        await navigateTo.shopNowPage()
+
+        //navigate to Bedding page
+        await navigateTo.beddingPage()
+
+        //Click on the White Linen Duvet Cover 
+        await onBeddingPage.clickOnWhiteLinenDuvetCover()
+
+        //price before cart page 
+        const initialPrice = await page.getByText('$').innerText()
+
+        // Removing the $ sign
+        const priceWithout$sign = initialPrice.replace(/[^\d.]/g, '');
+
+        // Parsing the string into a number
+        const parseValue = parseFloat(priceWithout$sign);
+
+        // The value of priceElementValue multiplied by 2
+        const initialValueMultiple2 = parseValue * 2;
```

Comment:
You have misspelling word in the variable name. 'Multiple' should be 'Multiply' and the whole name can be 'initialValueMultipliedBy2'